example/App.vue

```diff
@@ -66,6 +66,7 @@
     <div class="btn-toggle">
       <button @click="activateFullScreenGreenBg()">Full Screen (Green background color)</button>
     </div>
+    <input class="btn-toggle" type="text" v-model="text" placeholder="custom text "></input>
```

Comment:
- Please remove a space after `custom text`.
- Input tag should be `<input/>`, not `<input></input>`.